CONTRIBUTING.md

```diff
@@ -34,17 +34,17 @@ Before sending your pull requests, make sure you do the following:
   reviewer.
 - If the PR didn't meet the validation criteria, we request for additional
   changes to be made to PR to pass quality checks and send it back or on a rare
-  occassion we may reject it.
+  occasion we may reject it.
 
 **3. Review**
 
-- For Valid PR, reviewer (person familiar with the code/functionality) checks if
+- For a valid PR, reviewer (person familiar with the code/functionality) checks if
   the PR looks good or needs additional changes.
-- If all looks good, reviewer would approve the PR.
-- If a change is needed, the contributor is requested to make suggested change.
-- You make the change and submit for the review again.
-- This cycle repeats itself till the PR gets approved.
-- Note: As a friendly reminder we may reach out to you if the PR is awaiting
+- If all looks good, the reviewer will approve the PR.
+- If a change is needed, the contributor is requested to make the suggested change.
+- You make the change and submit it for the review again.
+- This cycle repeats itself untill the PR gets approved.
```

Comment:
I think there is still a typo in this line, you accidentally spelled 'until' with 2 'l's instead of one.